gslib/commands/mb.py

```diff
@@ -147,11 +147,12 @@
                          "enforced", objects in this bucket cannot be made
                          publicly accessible. Default is "unspecified".
 
-  --rpo setting          Specifies the `replication setting <https://cloud.google.com/storage/docs/turbo-replication>`_.
-                         Valid values are {rpo_values}. If unspecified,
-                         the DEFAULT setting is applied for dual-region and
-                         multi-region buckets and
-                         None for single-region buckets.
+  --rpo setting          Specifies the replication setting <https://cloud.google.com/storage/docs/turbo-replication>_.
```

Comment:
I think you'll still need the ` tickmarks for the linking to work?

Reply:
Good catch!